cmd/image-builder/github-workflow-integration.md

```diff
@@ -0,0 +1,156 @@
+   service to sign the OCI image.
+   This step ensures the integrity and authenticity of the OCI image.
+
+## GitHub OIDC Identity Token Claims
+
+The OIDC token issued by GitHub's OIDC identity provider contains several claims that are crucial for the `oci-image-builder` pipeline.
+These claims are used to identify the workflow triggering the build pipeline and to clone the appropriate version of the source code. This
+is essential for slc-29 compliance, as it ensures that the exact version of the code that was tested in the PR or for which the push was
+merged is built.
+
+The validity and integrity of OIDC token must be validated in `oci-image-builder` pipeline and fail pipeline execution if validation fails.
+Because OIDC token uses `JWT` format, it can be validated with a standard validation process against GitHub OIDC identity provider.
+
+### Workflow Identification Claims
+
+The OIDC token contains the following claims that can be used to identify the workflow that triggered the build pipeline. These include:
+
+- `iss`: The issuer of the token. This is always `https://token.actions.githubusercontent.com`.
+- `iat`: The time at which the token was issued.
+- `exp`: The time at which the token expires.
+- `jti`: A unique identifier for the token.
+- `nbf`: The time before which the token must not be accepted for processing.
+- `kid`: The key ID of the key used to sign the token.
+- `alg`: The algorithm used to sign the token.
+- `run_id`: The ID of the workflow run.
+- `run_number`: The number of the workflow run.
+- `actor`: The login of the user who initiated the workflow run.
+- `event_name`: The name of the event that triggered the workflow run.
+- `workflow`: The name of the workflow that triggered the workflow run.
+- `workflow_ref`: The git ref associated with the workflow file.
+- `repository`: The repository where the workflow run occurred.
+- `repository_owner`: The owner of the repository where the workflow run occurred.
+
+### Source Code Cloning Claims
+
+The OIDC token also contains claims that can be used to clone the appropriate version of the source code:
+
+- `repository`: The repository where the workflow run occurred.
+- `repository_owner`: The owner of the repository where the workflow run occurred.
+- `event_name`: The name of the event that triggered the workflow run.
+- `ref`: The git ref associated with the workflow run. This can be used to checkout the correct branch, tag, or commit.
+- `ref_type`: The type of git ref associated with the workflow run. This can be used to determine whether the ref is a branch, tag, or
+  commit.
+- `base_ref`: The base git ref associated with the workflow run. This can be used to determine the base branch for a pull request.
+- `head_ref`: The head git ref associated with the workflow run. This can be used to determine the head branch for a pull request.
```

Comment:
If you can provide values for these claims, please use bold font for them (like you do for [parameters and variables](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#code-font-and-bold-font)).

Reply:
What I meant is, that claims are similar to parameters or variables In the sense that they can have some values (no matter where the values come from). Hence, they should be formatted in the same way, i.e. in bold font.